src/test/resources/ConnectionParameters.properties

```diff
@@ -1,9 +1,7 @@
 DownloadPath=/Users/ksahil/Downloads/
 
-#My username and Passwords
-gLoginUsername=ksahil@google.com
-gPassword=*********
-
+Project-ID=cdf-athena
+windowSize=--window-size=1920x1040
```

Comment:
Can you just have `1920x1040` here and have `--window-size=`  in the code?

Reply:
Done